Resources/doc/reference/action_export.rst

```diff
@@ -1,31 +1,98 @@
 The Export action
 =================
 
-.. note::
-
-    This document is a stub representing a new work in progress. If you're reading
-    this you can help contribute, **no matter what your experience level with Sonata
-    is**. Check out the `issues on GitHub`_ for more information about how to get involved.
-
 This document will cover the Export action and related configuration options.
 
 Basic configuration
 -------------------
 
+If you have registered the ``SonataExporterBundle`` bundle in the kernel of your application,
+you can benefit from a lot of flexibility:
+
+* You can configure default exporters globally.
+* You can add custom exporters, also globally.
+* You can configure every default writer.
+
+See `the exporter bundle documentation`_ for more information.
+
 Translation
 ~~~~~~~~~~~
 
 All field names are translated by default.
 An internal mechanism checks if a field matching the translator strategy label exists in the current translation file
 and will use the field name as a fallback.
 
+Picking which fields to export
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, all fields are exported. More accurately, it depends on the
+persistence backend you are using, but for instance, the doctrine ORM backend
+export all fields (associations are not exported). If you want to change this
+behavior for a specific admin, you can override the ``getExportFields()`` method:
```

Comment:
You should use `configureExportFields`, because the `getExportFields()` should be made final. All modifications should be handled in the `configure...` methods.